docs/manual/releasing.rst

```diff
@@ -0,0 +1,47 @@
+===============
+Releasing Qtile
+===============
+
+Here is a short document about how to tag Qtile releases.
+
+1. create a "Release vX.XX.XX" commit. I like to thank all the contributors; I
+   find that list with something like:
+
+.. bash::
+
+   git log --oneline --no-merges --format=%an $(git describe --tags --abbrev=0)..HEAD | sort -u
+
+note that this can sometimes generate duplicates if people commit with slightly
+different names; I typically clean it up manually.
+
+Be sure that you GPG-sign (i.e. the ``-S`` argument to ``git commit``) this commit.
+
+2. Create a GPG-signed annotated tag (``git tag -a -s vX.XX.XX``); I usually just use
+   exactly the same commit message as the actual release commit above.
+
+3. Push your tag to qtile/qtile directly:
+
+.. bash::
+
+   git push origin vX.XX.XX
+
+4. Check the `Release action
+   <https://github.com/qtile/qtile/actions/workflows/release.yml>`_; the "Test
+   PyPI upload" action should build and upload the wheels to the test
+   environment. If this step breaks, you can delete your tag (``git push origin
+   :vX.XX.XX``), fix the bug, re-tag, and hopefully that will work.
+
+5. If the release worked, push to qtile/qtile master directly (``git push
+   origin master``).
```

Comment:
What are you pushing here? I don't think you want any commits being added after the tag. So, once you've fixed the bug, re-tagged the commit, test release builds ok (all step 4) then you go to step 6.

Reply:
I want the release commits on the git master branch. All the release fixing commits should really go *before* this release commit. Perhaps it's better to delete all of this language, as it's too confusing.

Reply:
Yes. I don't think you need step 5.